lib/api/fields/types/textarea.php

```diff
@@ -22,7 +22,8 @@
  * }
  */
 function beans_field_textarea( array $field ) {
-	printf( '<textarea id="%s" name="%s" %s>%s</textarea>',
+	printf(
+		'<textarea id="%s" name="%s" %s>%s</textarea>',
 		esc_attr( $field['id'] ),
 		esc_attr( $field['name'] ),
 		beans_esc_attributes( $field['attributes'] ), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaping is handled in the function.
```

Comment:
Looks like we have to add the comment above the line in this case. (Might be a bug in phpcs...)
```
// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaping is handled in the function.
beans_esc_attributes( $field['attributes'] ),
```